File: Code/05_File_POSIX/Ex1.c
#include <stdio.h>	// Para a funcao printf()
#include <stdlib.h>	// Para a função exit()
#include <fcntl.h>	// Para a funcao open()
#include <unistd.h>	// Para a funcao close()

int main(int argc, const char * argv[])
{
	int fd = open("exemplo.bin", O_RDWR);
	if(fd==-1)
	{
		printf("Erro na abertura"
			" do arquivo.\n");
		exit(-1);
	}
	close(fd);

	return 0;
}

File: 1_Introdução/1.4_GCC/Ex1/Ex1.py
import sys

print("argc = %d" % len(sys.argv))
print("arg[0]: " + sys.argv[0])
if(len(sys.argv)>1):
	print("arg[1]: " + sys.argv[1])
if(len(sys.argv)>2):
	print("arg[2]: " + sys.argv[2])
